### PR TITLE
seed with hammer csv

### PR DESCRIPTION
**Pull request by thomasmckay**
An example of seeding with hammer csv. 

- [ ] install hammer locally on server
- [ ] add a readme describing usage and variables
- [x] customize csv file location
- [x] if csv resource type file does not exist, don't run (skip some resources)
- [x] if csv resource type file already exists, don't overwrite (allow other roles to put in custom files)
- [x] capture stderr and stdout to logs

### CODE REVIEW

**Comment by ekohl** [comment]
Would it be possible to seed through json instead of csv? I think that would make is easier to reuse the logic for other types.

**Comment by thomasmckay** [comment]
@ekohl The "logic" to import is all written in hammer csv. I suspect you could add some ansible magic to take in a json and convert it into the "with_items" line that would then generate the csv for import.

I didn't include the other resource types but they are listed here https://github.com/Katello/hammer-cli-csv/blob/master/lib/hammer_cli_csv.rb . They are not all fully supported or working. Maybe hooking into forklift would provide an incentive for making it more robust for all the other resource types.

**Comment by thomasmckay** [comment]
I'll add the rest of the resource types to this PR for completeness.

File: playbooks/roles/csv/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    owner: vagrant
+
+- name: Run hammer csv operating-systems
+  shell: >
+    BUNDLE_GEMFILE=~/hammer-cli-katello/Gemfile bundle exec {{ csv_hammer }}
+    csv operating-systems
+    --verbose
+    --file /home/vagrant/csv/operating-systems.csv
+    > /home/vagrant/csv/operating-systems.log
+
```
Comment (thomasmckay) [inline playbooks/roles/csv/tasks/main.yml]: Is there some way to put the above two sections into a loop that iterates over all the resource types? Would be better than cut&paste of this section.

File: .gitignore [hunk 2]
```diff
@@ -4,7 +4,6 @@
 *.swn
 katello-installer*
 foreman-installer*
-puppet-*
```
Comment (thomasmckay) [inline .gitignore]: @ehelms This line matched one of my file templates; is there a more specific pattern to ignore? Or I could force not ignoring my file.

Comment (ehelms) [inline .gitignore]: I think this removal is fine now.

**Comment by ehelms** [comment]
@thomasmckay Hallo! This PR recently celebrated it's birthday. Do you plan to fix it up or should we close it out?

**Pull request closed without merging**